Consul.Test/AgentTest.cs

```diff
@@ -993,5 +993,12 @@ public async Task Agent_FilterChecks()
             Assert.Single(actual);
             Assert.Equal(checkName, actual.Values.First().Name);
         }
+
+        [Fact]
+        public async Task Agent_Metrics()
+        {
+            var agentMetrics = await _client.Agent.GetAgentMetrics();
+            Assert.NotNull(agentMetrics.Response.Timestamp);
```

Comment:
I don't see a reason why not to assert, that all other members are not null:
```suggestion
            Assert.NotNull(agentMetrics.Response.Timestamp);
            Assert.NotNull(agentMetrics.Response.Counters);
            Assert.NotNull(agentMetrics.Response.Gauges);
            Assert.NotNull(agentMetrics.Response.Points);
            Assert.NotNull(agentMetrics.Response.Samples);
```